upstream-community-operators/appdynamics/0.4.1/appdynamics-operator.v0.4.1.clusterserviceversion.yaml

```
@@ -0,0 +1,219 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "appdynamics.com/v1alpha1",
+          "kind": "Adam",
+          "metadata": {
+            "name": "example-adam",
+            "namespace": "appdynamics"
+          },
+          "spec": {
+          "controllerUrl": "https://saas.appdynamics.com"
+          }
+        },
+        {
+          "apiVersion": "appdynamics.com/v1alpha1",
+          "kind": "Clusteragent",
+          "metadata": {
+            "name": "k8s-cluster-agent",
+            "namespace": "appdynamics"
+          },
+          "spec": {
+            "account": "customer1",
+            "appName": "Cluster1",
+            "controllerUrl": "https://saas.appdynamics.com",
+            "stdoutLogging": "true"
+          }
+        },
+        {
+          "apiVersion": "appdynamics.com/v1alpha1",
+          "kind": "InfraViz",
+          "metadata": {
+            "name": "appd-infraviz",
+            "namespace": "appdynamics"
+          },
+          "spec": {
+            "account": "customer1",
+            "controllerUrl": "https://saas.appdynamics.com",
+            "enableDockerViz": "false",
+            "enableMasters": true,
+            "globalAccount": "customer1_12345",
+            "netVizPort": 3892,
+            "stdoutLogging": true
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Monitoring
+    certified: 'false'
+    createdAt: 2019-11-30 12:00:00
+    containerImage: docker.io/appdynamics/cluster-agent-operator
+    description: End to end monitoring of applications on Kubernetes and OpenShift clusters with AppDynamics.
+    support: support@appdynamics.com
+    repository: https://github.com/Appdynamics/appdynamics-operator
+  name: appdynamics-operator.v0.4.0
```

```suggestion
  name: appdynamics-operator.v0.4.1
```

Since the version is not what the package is pointing to, this package fails to load into the catalog.